packages/webapp/src/pages/ProjectSettings.tsx

```diff
@@ -207,10 +212,11 @@ export default function ProjectSettings() {
     };
 
     const handleRemoveEnvVariable = async (index: number) => {
+        console.log(envVariables);
```

Comment:
should this be removed?

Reply:
Thanks, https://github.com/NangoHQ/nango/pull/1351/commits/206072616b8bc4862b2c836d1f278e97d1610ec4